test/extended/util/cli.go

```diff
@@ -233,22 +234,21 @@ func (c *CLI) printCmd() string {
 }
 
 // Output executes the command and return the output as string
-func (c *CLI) Output() (out string, err error) {
+func (c *CLI) Output() (string, error) {
 	if c.verbose {
 		fmt.Printf("DEBUG: oc %s\n", c.printCmd())
 	}
-	// Capture the panic and convert it to a regular error
-	defer func() {
-		if r := recover(); r != nil {
-			out = fmt.Sprintf("%s", r)
-			err = fmt.Errorf("PANIC: %s", out)
-		}
-	}()
-	err = c.cmd.Execute()
+	out, err := exec.Command(c.execPath, c.finalArgs...).CombinedOutput()
+	trimmed := strings.TrimSpace(string(out))
+	if err != nil {
+		e2e.Logf("ERROR:\n%s", trimmed)
+		FatalErr(err)
```

Comment:
also does this mean we FatalErr when command exists != 0 status? I think we just want to return error in that case (to support error cases)